discord-lambda/register_endpoints.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+DISCORD_CLIENT_ID = os.getenv('DISCORD_CLIENT_ID')
+DISCORD_GUILD_ID = os.getenv('DISCORD_GUILD_ID')
```

Comment:
The whole "guild" thing keeps taking me aback -- very much Discord showing its roots.